src/main/java/com/rocketden/main/service/UserService.java

```diff
@@ -37,6 +38,19 @@ public UserDto createUser(CreateUserRequest request) {
         return UserMapper.toDto(user);
     }
 
+    public UserDto deleteUser(DeleteUserRequest request) {
+        User user = repository.findUserByNickname(request.getNickname());
```

Comment:
If there are multiple users with the given nickname, does this just get the first user?

Reply:
Yes, since right now UserDto doesn't contain another ID. This depends on how findUserByNickname works though which I'm not super clear about.